features/environment.py

```diff
@@ -276,6 +276,15 @@ def terminate_backends(self):
             if 'process' not in p.cmdline()[0]:
                 p.terminate()
 
+    @property
+    def backup_source(self):
+        return 'postgres://{username}:{password}@{host}:{port}/{database}'.format(**self._replication)
+
+    def backup(self, dest='basebackup'):
```

Comment:
I've been thinking about it, but unfortunately it will require do define a global variable.